docs/change-log/index.md

```diff
@@ -13,16 +14,185 @@
+The new service is transparent to the end-user: there are no UI or API changes. 
+
+### Breaking changes
+
+- The behavior of ingest with aggregation changed in v1.3.0 (storey, spark, pandas engines). Now, when you ingest a "timestamp" column, it returns <br>
+`<class 'pandas._libs.tslibs.timestamps.Timestamp'>`. <br>Previously, it returned `<class 'str'>`
+
+- Any target data that was saved using Redis as an online target with storey engine (RedisNoSql target, introduced in 1.2.1) is not accessible after upgrading to v1.3. (Data ingested subsequent to the upgrade is unaffacted.)
+
+
+
+<a id="api-130"></a>
+### Deprecated and removed APIs
+Starting with v1.3.0, and continuing in subsequent releases, obsolete functions are getting removed from the code.
+
+**Deprecated and removed from v1.3.0 code**<br>
+These MLRun APIs have been deprecated since at least v1.0.0 and were removed from the code:
+
+| Deprecated/removed                   | Use instead                                   |
+| ------------------------------------ | --------------------------------------------- |
+| `project.functions`                  | `project.get_functions`, `project.set_functions`, `project.list_functions` |
+| `project.artifacts`                  | `project.get_artifacts`, `project.set_artifacts`, `project.list_artifacts` |
+| `project.func()`                     | `project.get_function()`                       |
+| `project.create_vault_secrets()`     | NA                                   |
+| `project.get_vault_secret()`         | NA                                   |
+| `MlrunProjectLegacy` class           | `MlrunProject`                                 |
+| Feature-store: usage of state in graph. For example: `add_writer_state`, and the `after_state` parameter in `_init_` methods.  | `step`                        |
+| `mount_path` parameter in mount_v3io() | `volume_mounts`                        |
+| `NewTask`                            | `run_function()`                        |
+
+
+
+**Deprecated APIs, will be removed in v1.5.0**<br>
+These APIs will be removed from the v1.5.0 code. A FutureWarning appears if you try to use them in v1.3.0.
+| Deprecated / to be removed                       | Use instead                                   |
+| ------------------------------------------------ | --------------------------------------------- |
+| project-related parameters of `set_environment`. (Global-related parameters will not be deprecated.) | Project APIs 
+such as `get_or_create_project`, `load_project` |
+| `KubeResource.gpus`                              | `with_limits`                 |
+| Dask `gpus`                                      | `with_scheduler_limits` / `with_worker_limits`   |
+| Dask `with_limits`                               | `with_scheduler_limits` / `with_worker_limits`    |
+| Dask `with_requests`                             | `with_scheduler_requests` / `with_worker_requests`    |
```

Comment:
both were removed in previous versions.
see https://jira.iguazeng.com/browse/ML-3104 (Deprecated since MLRun 0.8.0)

Reply:
moved to preceding section